docker/README.md

```diff
@@ -0,0 +1,122 @@
+# Celestia `txsim` Docker Image Usage Guide
+
+The `txsim` binary is a tool that can be used to simulate transactions on the Celestia network. It can be used to test the performance of the Celestia network.
+This guide provides instructions on how to use the Celestia `txsim` Docker image. The `txsim` Docker image is designed to run the `txsim` binary with a variety of configurable options.
+
+# Table of Contents
+
+1. [Prerequisites](#prerequisites)
+2. [Running the Docker Image](#running-the-docker-image)
+    - [Docker Run](#docker-run)
+    - [Docker Compose](#docker-compose)
+    - [Kubernetes Deployment](#kubernetes-deployment)
+3. [Flag Breakdown](#flag-breakdown)
+
+## Prerequisites
+
+Before you can use the `txsim` Docker image, you must have a prefunded account set up. The `txsim` binary requires a prefunded account to function correctly. The keyring for this account should be stored in a file that can be accessed by the Docker container.
+
+## Running the Docker Image
+
+### Docker Run
+
+You can run the `txsim` Docker image using the `docker run` command. Here's an example:
+
+```bash
+docker run -it -v ${HOME}/.celestia-app:/home/celestia ghcr.io/celestiaorg/txsim -k 0 -r http://consensus-validator-robusta-rc6.celestia-robusta.com:26657,http://consensus-full-robusta-rc6.celestia-robusta.com:26657 -g consensus-validator-robusta-rc6.celestia-robusta.com:9090 -t 10s -b 10 -d 100 -e 10
```

Comment:
[optional] I'm a Docker noob so I don't know what these shorthand flags do. The long format flags would help a lot for readability

Motivation: https://changelog.com/posts/use-long-flags-when-scripting

For example `--volume` instead of `-v`

Reply:
If I get rid of getopts, then yes, we will have long-format flags. 
But I want to pursue a simple server approach that can be triggered rather on/off rather than a cli command 😅